db/seeds.json

```diff
@@ -0,0 +1,9 @@
+compliments = [
+  "Your instructors love you",
+  "High five = ^5",
+  "Is it Ruby Tuesday yet?",
+  "It's almost beer o'clock",
+  "The Force is strong with you"
+]
+
+colors = ["#FFBF00", "#0080FF","#01DF3A","#FF0080"]
```

Comment:
Your seed file isn't written in valid JSON. It should look something like this...

```json
{
  "compliments": [
    "Your instructors love you",
    "High five = ^5",
    "Is it Ruby Tuesday yet?",
    "It's almost beer o'clock",
    "The Force is strong with you"
  ],
  "colors": [
    "#FFBF00",
    "#0080FF",
    "#01DF3A",
    "#FF0080"
  ]
}
```